packages/lib/responses.ts

```diff
@@ -1,40 +1,52 @@
 import { TResponse } from "@formbricks/types/responses";
-import { TSurvey, TSurveyQuestionType } from "@formbricks/types/surveys";
+import { TSurvey, TSurveyQuestion, TSurveyQuestionType } from "@formbricks/types/surveys";
 
 import { getLocalizedValue } from "./i18n/utils";
 
+export const getResponseValue = (
```

Comment:
please add a comment what this function is doing and also a return value. I'm a bit confused since it's named `getResponseValue` and is passed an `answer`.
`getAnswerFromResponse` would sound more explaining; but it's already passed the answer, so I'm confused.